src/registerServiceWorker.js

```diff
@@ -0,0 +1,108 @@
+// In production, we register a service worker to serve assets from local cache.
+
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on the "N+1" visit to a page, since previously
+// cached resources are updated in the background.
+
+// To learn more about the benefits of this model, read https://goo.gl/KwvDNy.
```

Comment:
@sirkitree This comes standard with `react-scripts`, it doesn't require adjustment or troubleshooting, however I find it strange that react-scripts hasn't moved it into a package.. I'm completely OK with removing this, I don't think it's necessarily beneficial in this case.